src/mnemonics/german.h

```diff
@@ -1678,11 +1678,11 @@ namespace Language
         "Zündung",
         "Zweck",
         "Zyklop"
-      }), 4)
-    {
+      }), 4)
+    {

```

Comment:
An easy way is:
git reset HEAD^ (this "uncommits" the top commit)
git add -p src/mnemonics/german.h
Select which of the hunks you want to commit or not
Do this with the rest of the files that changed
git commit (or git commit -S if you sign your commits)